packages/app/src/theme/aperture.ts

```diff
@@ -257,10 +257,10 @@ const createCustomThemeOverrides = (
   };
 };
 
-export const apertureTheme: BackstageTheme = {
+export const apertureTheme: UnifiedTheme = {
```

Comment:
Struggling with getting this working. Could use your help @timbonicus when you have some time, please